node/lib/util/stitch_util.js

```diff
@@ -804,6 +811,14 @@ exports.makeGetConvertedCommit = function (repo, cache) {
             return cache[sha];
         }
         const result = yield exports.readConvertedCommit(repo, sha);
+
+        try {
+            yield repo.getCommit(result);
+        } catch (e) {
+            // We have the note but not the commit; treat as missing
+            return undefined;
```

Comment:
Will it be a problem if those commits are not converted? I think they maybe pushed later, right?

Reply:
We'll just reconvert them.